spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -372,36 +347,26 @@ def check_response_error(code)
   # POST
 
   context 'POST /authorship' do
+    let(:http_request) { post '/authorship', request_data.to_json, headers }
+
     context 'success' do
-      let(:http_request) do
-        post '/authorship', request_data.to_json, headers
-        expect(response.status).to eq(201)
-      end
+      after(:example) { expect(response.status).to eq(201) }
```

Comment:
The shared example(s) now checks for success (right?), so does this `after` block now check it again?

Reply:
Yeah, this is more specific (201 is one flavor of success).